src/test/snapshot/snapshot.md

```diff
@@ -343,6 +343,55 @@ this is just text i guess
 ---
 
 
+==========
+015 Code blocks.md
+==========
+---
+{"dg-publish":true,"permalink":"/015-code-blocks/"}
+---
+
+These codeblocks should not be modified upon publish.
+
+Sample 1
+```jinja2
+{{ highlight_text }}{% if highlight_location and highlight_location_url %} ([via]({{highlight_location_url}})){% elif highlight_location %} ({{highlight_location}}){% endif %} ^rwhi{{highlight_id}}
+{% if highlight_note %}
+{{ highlight_note }} ^rwhi{{highlight_id}}-note
+{% endif %}
+```
+
+Sample 2
+```md
+In medieval Latin a florilegium (plural florilegia) was a compilation of excerpts from other writings.
+ The word is from the Latin flos (flower) and legere (to gather): literally a gathering of flowers, or collection of fine extracts from the body of a larger work. ([via](https://en.wikipedia.org/wiki/Florilegium)) ^rwhi724352030
+```
+
+Sample 3
+```
+This codeblock has a transclusion syntax in it.
+Check it out: 
+<div class="transclusion internal-embed is-loaded"><a class="markdown-embed-link" href="/001-links/" aria-label="Open link"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon lucide-link"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path></svg></a><div class="markdown-embed">
```

Comment:
Just noting that this is actually evidence of a very similar bug: https://github.com/oleeskild/obsidian-digital-garden/issues/599